ui/StatusQ/tests/TestCore/TestModels/tst_ExpressionRole.qml

```diff
@@ -0,0 +1,130 @@
+import QtQml 2.15
+import QtQuick 2.15
+import QtTest 1.15
+
+import SortFilterProxyModel 0.2
+
+import StatusQ 0.1
+import StatusQ.Core.Utils 0.1
+
+import StatusQ.TestHelpers 0.1
+
+Item {
+    id: root
+
+    Component {
+        id: testComponent
+
+        QtObject {
+            property int d: 0
+
+            readonly property ListModel source: ListModel {
+                id: listModel
+
+                ListElement { a: 1; b: 2; c: 3 }
```

Comment:
maybe we add some tests for nested models too? in case we can. wdyt?

Reply:
Basically, the type doesn't matter here, double/int/string/model or any other type supported by qvariant should be ok.